src/compressor/CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ if(HAVE_BROTLI)
   add_subdirectory(brotli)
 endif()
 
+add_library(compressor STATIC ${compressor_srcs})
```

Comment:
who is supposed to consume this library?

Reply:
none. it is like "add_library(erasure_code STATIC ErasureCodePlugin.cc)" in erasure framework, looks like I didn't find any consumer of erasure_code library.

Reply:
https://github.com/ceph/ceph/blob/69c70bd96f6481175ceaab4fc09416ca9febc0ad/src/CMakeLists.txt#L401-L402

Reply:
thanks, maybe compressor can be also added into this place.